app/templates/macros.html

```diff
@@ -76,9 +76,12 @@
     <div class="btn btn-choose">Choose File</div>
     <div class="filename-wrapper">
         {{ render_field(form.filename, label=False, placeholder="No File Chosen", required="")}}
+        <div id='ajax-loading-images'>
```

Comment:
Why do we have this as well as the `<div id='images-ajax-loadpoint'> ` in `app/templates/admin/events/create.html`?